stanford_profile.profile

```diff
@@ -21,3 +27,50 @@ function stanford_profile_install_tasks(&$install_state) {
 function stanford_profile_final_task(array &$install_state) {
   \Drupal::service('plugin.manager.install_tasks')->runTasks($install_state);
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function stanford_profile_preprocess(array &$variables, $hook) {
+
+  $variables['su_use_theme_logo'] = '1';
```

Comment:
This would be a good place to leave a code comment on what is going on and why.

Reply:
ok, the default is to use the theme logo, so this sets the variable that the twig template uses. if the config page has not been saved. this sets the default setting.